vendor/roc_std/src/roc_list.rs

```diff
@@ -170,7 +182,7 @@ where
 
         let new_len = self.len() + slice.len();
         let non_null_elements = if let Some((elements, storage)) = self.elements_and_storage() {
-            // Decrement the list's reference count.
+            // Decrement the list's refence count.
```

Comment:
This is a typo, but I assume this is generated code an really a roc glue bug?

Reply:
Hah, well spotted. Looks like it's in roc_std, actually!